document/core/syntax/values.rst

```diff
@@ -108,7 +108,7 @@ No distinction is made between signalling and quiet NaNs.
      {+} \fNmag ~|~ {-} \fNmag \\
    \production{floating-point magnitude} & \fNmag &::=&
      (1 + \uM\cdot 2^{-M}) \cdot 2^e & (\iff -2^{E-1}+2 \leq e \leq 2^{E-1}-1) \\ &&|&
-     (0 + \uM\cdot 2^{-M}) \cdot 2^e & (\iff e = -2^{E-1}+2) \\ &&|&
+     (0 + \uM\cdot 2^{-M}) \cdot 2^e & (\iff e = -2^{E-1}+1) \\ &&|&
```

Comment:
I believe this was correct before: in the abstract syntax, subnormals are distinguished by the leading 0, while the exponent has the same value as the smallest possible normal one (it is, however, encoded specially, see the binary format). This may be worth adding a note for.